message-queue-lib/project.clj

```diff
@@ -18,8 +18,9 @@
   :dependencies [[cheshire "5.8.1"]
                  [clj-http "2.3.0"]
                  [clj-time "0.15.1"]
-                 [com.amazonaws/aws-java-sdk-sns ~aws-java-sdk-version]
+                 [com.amazonaws/aws-java-sdk-sns ~aws-java-sdk-version :exclusions [com.fasterxml.jackson.core/jackson-databind]]
```

Comment:
nit: long line

Reply:
Do we have a preferred style here? exclusion on its own line or with the library?:
```
[com.amazonaws/aws-java-sdk-sns ~aws-java-sdk-version
 :exclusions [com.fasterxml.jackson.core/jackson-databind]]
```

Versus:
```
[com.amazonaws/aws-java-sdk-sns ~aws-java-sdk-version
 :exclusions 
 [com.fasterxml.jackson.core/jackson-databind]]
```